environment.yml

```diff
@@ -8,3 +8,9 @@ dependencies:
   - requests
   - conda-smithy
   - pygithub
+#  - cirun
+#  - conda-smithy
+# TEMP Remove pip install after testing
+  - pip:
+    - git+https://github.com/jaimergp/conda-smithy.git@gha-self-hosted-cirun
+    - git+https://github.com/aktechlabs/cirun-py.git@main
```

Comment:
We need to release these two before merging. Can you get started with `cirun-py` or are you expecting more changes there?